server/src/server.ts

```diff
@@ -32,13 +33,15 @@ app.use(
     secret: process.env.SERVER_SESSION_SECRET!,
     resave: false,
     saveUninitialized: true,
-    store: memoryStore,
+    store: new MemoryStore({
+      checkPeriod: 86400000, // prune expired entries every 24h
+    }),
   })
 );
 
 const keycloak = new Keycloak(
   {
-    store: memoryStore,
+    store: MemoryStore,
```

Comment:
This is the memory store constructor, not the object. Both the session and Keycloak should use the same memory store as before.